### ISSUE TITLE
Dots on top of each other

### ISSUE THREAD

**Issue by BillPascoe** [HughCraig/GHAP#132]
Journey view dots on top of each other obscure each other. Ideally would be exploding spider view.

**Comment by MufengNiu** [comment]
Hi @BillPascoe, By exploding spider view, do you mean something like : 
<img width="315" alt="Screenshot 2023-11-08 at 12 52 14" src="https://github.com/HughCraig/GHAP/assets/51284623/b7c39740-e113-4218-a19a-3977109afcba">

**Comment by BillPascoe** [comment]
Yes something like that. I think Leaflet maps have it but I don't think ArcGIS JS does, but worth looking into. If it is not possible, see if we can do journey views with clusters. Here's an examples of 'spiderfy': 
![spiderfy2](https://github.com/HughCraig/GHAP/assets/27399676/45208abb-9d6f-4467-bcca-dfef0ea6d36a)
